src/transform.ts

```diff
@@ -97,6 +98,57 @@ export interface AggregatedFieldDef {
   as: string;
 }
 
+export interface WindowFieldDef {
+  /**
+   * The operations supported for the window aggregation
+   */
+  op: AggregateOp | WindowOnlyOp;
+
+  /**
+   * The optional parameters for the operation
```

Comment:
Add full stop. 

Explain a bit more that look at the operator table to see parameter for each operation.

Reply:
Resolved.